src/lib/components/filters/content.svelte

```diff
@@ -85,7 +85,7 @@
 </script>
 
 <div>
-    <form on:submit|preventDefault={addFilterAndReset}>
+    <form onsubmit={addFilterAndReset}>
```

Comment:
We don't need prevent default?

Reply:
added it manually anyways, just for safety! its just the new syntax doesn't like things being piped in there.